src/components/Validators/ValidatorStats.vue

```diff
@@ -0,0 +1,260 @@
+                ></v-progress-circular>
+            </div>
+        </article>
+    </div>
+</template>
+
+<script lang="ts">
+import 'reflect-metadata'
+import { Vue, Component } from 'vue-property-decorator'
+import TooltipMeta from '@/components/Home/TopInfo/TooltipMeta.vue'
+import { stringToBig, bigToDenomBig } from '@/helper'
+import Big from 'big.js'
+import { TOTAL_AVAX_SUPPLY } from '@/store/modules/platform/platform'
+import { avalanche } from '@/avalanche'
+import { Defaults, ONEAVAX } from 'avalanche/dist/utils'
+import { BN } from 'avalanche/dist'
+
+@Component({
+    components: {
+        TooltipMeta,
+    },
+})
+export default class ValidatorStats extends Vue {
+    get subnetsLoaded(): boolean {
+        return this.$store.state.Platform.subnetsLoaded
+    }
+
+    // Data from Avalanche-Go
+    get totalStake(): string {
+        let res = this.$store.getters['Platform/totalStake']
+        res = stringToBig(res.toString(), 9).toFixed(0)
+        return parseInt(res).toLocaleString()
+    }
+
+    get percentStaked(): string {
+        let totalStake = this.$store.getters['Platform/totalStake']
+        totalStake = bigToDenomBig(totalStake, 9)
+        const percentStaked = totalStake.div(TOTAL_AVAX_SUPPLY).times(100)
+        return percentStaked.toFixed(2)
+    }
+
+    get annualStakingRewardPercentage(): string {
+        const networkID = avalanche.getNetworkID()
+
+        //@ts-ignore
+        const defaultValues = Defaults.network[networkID]
+        if (!defaultValues) {
+            console.error('Network default values not found.')
+            return 'TBD'
+        }
+
+        const maxConsumption: number = defaultValues.P.maxConsumption
+        const minConsumption: number = defaultValues.P.minConsumption
+        const avgReturn: number = ((maxConsumption + minConsumption) / 2) * 100
+
+        return `${avgReturn.toFixed(0)}%`
+    }
+
+    get annualStakingReward(): BN {
+        const totalStake: string = this.$store.getters[
+            'Platform/totalStake'
+        ].toFixed()
+        const totalStake_BN: BN = new BN(totalStake)
+        const ONE_YEAR_SECONDS = 60 * 60 * 24 * 365
+        const TOTAL_AVAX_SUPPLY_BN = new BN(
+            TOTAL_AVAX_SUPPLY.times(Math.pow(10, 9)).toFixed()
+        )
+
+        const annualStakingReward: BN = this.calculateStakingReward(
+            totalStake_BN,
+            ONE_YEAR_SECONDS,
+            TOTAL_AVAX_SUPPLY_BN
+        )
+        return annualStakingReward
+    }
+
+    calculateStakingReward(
+        amount: BN,
+        duration: number,
+        currentSupply: BN
+    ): BN {
+        const networkID = avalanche.getNetworkID()
+
+        //@ts-ignore
+        let defValues = Defaults.network[networkID]
+
+        if (!defValues) {
+            console.error('Network default values not found.')
+            return new BN(0)
+        }
+        defValues = defValues.P
+
+        const maxConsumption: number = defValues.maxConsumption
```

Comment:
```suggestion
        const { maxConsumption, minConsumption,  maxStakingDuration } = defValues
```

Reply:
Try to stay away from strict typing when possible, ideally the type are inferred from the rot so you dont have type casting unintentionally